### DISCUSSION TITLE
Downloading and then deleting pictures/videos from iCloud

### DISCUSSION THREAD

**Discussion by LawMixer** [icloud-photos-downloader/icloud_photos_downloader#989] [Q&A] [answered]
Is there a way to download every picture/video I have and then once it's downloaded, deleting it to free up space in my iCloud?

I'm thinking of setting up a python script to automate this, but I'm not sure how to get started.

Thanks for creating this!

**Comment by AndreyNikiforov** [comment, marked as answer]
check [operation modes](https://icloud-photos-downloader.github.io/icloud_photos_downloader/mode.html)
